source/github-helpers/index.ts

```diff
@@ -24,12 +24,13 @@ Example tag content on private repositories https://github.com/private/private/c
 export const getCurrentBranch = (): string | undefined => {
 	// .last needed for #2799
 	const feedLink = select.last('link[type="application/atom+xml"]');
+	const findLink = pageDetect.isPRFiles() ? false : select.last('a[data-hotkey="t"]');
```

Comment:
.last is needed

Reply:
It looks like you should just be able to extend the existing selector, e.g.

```
const linkWithData = select.last('link[type="application/atom+xml"], a[data-hotkey="t"]');
```

Reply:
We can't do that because we need to return false on `isPrFiles` and I dont want to break GHE